Keas.Mvc/ClientApp/components/Keys/KeyContainer.tsx

```diff
@@ -267,6 +266,11 @@ export default class KeyContainer extends React.Component<IProps, IState> {
         return keyInfo;
     };
 
+    private _checkIfKeyCodeIsValid = (code: string) => {
+        const keyCodes = this.state.keys.map(x => x.key.code);
+        return keyCodes.indexOf(code) === -1;
```

Comment:
@laholstege this works fine, but you could do a one-liner w/ either .find or .some or .findIndex.  I think I'd do it something like `return this.state.keys.some(x => x.key.code === code)`